ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -167,6 +158,14 @@
       />
     {{/if}}
 
+    {{#if (can 'navigate model' @model collection='sessions')}}
```

Comment:
How come we moved sessions to the bottom of the list?

Reply:
It felt better there for me because when onboarding, you won't have any sessions until you configure your targets, hosts, credentials. So just chronologically, sessions come at the end. I think Sessions was first because we thought they would be monitoring sessions closely, but I don't think that's the case for most admins. Feel free to argue for it to stay first though!

Reply:
Nope that makes sense!